cli/cmd/root.go

```diff
@@ -337,4 +340,5 @@ func addProxyConfigFlags(cmd *cobra.Command, options *proxyConfigOptions) {
 	cmd.PersistentFlags().UintSliceVar(&options.ignoreInboundPorts, "skip-inbound-ports", options.ignoreInboundPorts, "Ports that should skip the proxy and send directly to the application")
 	cmd.PersistentFlags().UintSliceVar(&options.ignoreOutboundPorts, "skip-outbound-ports", options.ignoreOutboundPorts, "Outbound ports that should skip the proxy")
 	cmd.PersistentFlags().BoolVar(&options.disableExternalProfiles, "disable-external-profiles", options.disableExternalProfiles, "Disables service profiles for non-Kubernetes services")
+	cmd.PersistentFlags().BoolVar(&options.noInitContainer, "--linkerd-cni-enabled", options.noInitContainer, "Experimental: Omit the proxy-init container when injecting the proxy; requires the linkerd-cni plugin to already be installed")
```

Comment:
D'oh typo here: `"--linkerd-cni-enabled"` => `"linkerd-cni-enabled"`

Reply:
And likewise here, let's hide this install/inject flag for the time being. You can do that with:

```
	cmd.PersistentFlags().BoolVar(&options.noInitContainer, "linkerd-cni-enabled", options.noInitContainer, "Experimental: Omit the proxy-init container when injecting the proxy; requires the linkerd-cni plugin to already be installed")
	cmd.PersistentFlags().MarkHidden("linkerd-cni-enabled")
```

Reply:
@klingerf based on my testing on our cluster, the plugin is working as it should...it is just that our cluster configuration is...complicated. When the binaries and configurations are placed in the correct spots, it works as expected. That being said, I don't think that we need to hide the flag. This is ready to go.

Reply:
Ok, I will hide the flag per additional offline conversation. :)